packages/vscode-extension/src/commands/BaseAction.ts

```diff
@@ -150,6 +161,10 @@ export default abstract class BaseAction {
 	}
 
 	protected async runSuiteCloudCommand(args: { [key: string]: string | string[] } = {}, otherExecutionPath?: string) {
+		if (this.refreshAuthorizationPeformed) {
+			args.skipAuthorizationCheck = BaseAction.SKIP_AUTHORIZATION_CHECK_VALUE;
+		}
+
```

Comment:
We always want to have `args.skipAuthorizationCheck` so node-cli code never executes the check, it doesn't matter if the refresh authz has been performed or was not needed.

When working on node-cli story I thought we will be adding the `skipAuthorizationCheck` directly in the `src/core/SuiteCloudRunner.ts`

Reply:
You are correct. Please check changes.